### PR TITLE
Heritage 378 GraphQL Banner

### PR DESCRIPTION
**Pull request by JohnHeeryTNA**
Ticket URL: https://national-archives.atlassian.net/jira/software/c/projects/HERITAGE/issues/HERITAGE-378

## About these changes

adding in the graphQL banner to home page

## How to check these changes

Run branch locally and check banner against design.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [x] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: templates/home/home_page.html [hunk 1]
```diff
@@ -51,6 +51,24 @@ <h2 class="tna-heading tna-heading--l">
         </div>
 
     </div>
+
+<!--graph ql banner -->
+<div class="tna-container">
+
+    <div class="tna-column tna-column--width-1-5 tna-column--width-1-3-medium tna-column--full-small tna-column--full-tiny graph-banner-icon">
+        <img src ="/static/images/graph-ql-icon.jpg" alt="graphQL icon">
```
Comment (TNA-Allan) [inline templates/home/home_page.html:59]: Could not find this jpg file.

Comment (TNA-Allan) [inline templates/home/home_page.html:59]: Could not find  `/static/images/graph-ql-icon.jpg`

File: templates/home/home_page.html [hunk 2]
```diff
@@ -51,6 +51,24 @@ <h2 class="tna-heading tna-heading--l">
...
+    </div>
+    <div class="tna-column tna-column--width-3-4 tna-column--width-2-3-medium tna-column--full-small tna-column--full-tiny graph-banner">
+        <h3 class="tna-heading tna-heading--l">
+            <a href="">Query the data</a>
```
Comment (TNA-Allan) [inline templates/home/home_page.html:63]: The Graph QL end point is missing. Should the link open in a new tab/window?

File: templates/home/home_page.html (hunk 1 above)
Comment (JohnHeeryTNA) [reply templates/home/home_page.html:59]: Sorry, I had forgotten to add the file - should be there now.

File: templates/home/home_page.html (hunk 2 above)
Comment (JohnHeeryTNA) [reply templates/home/home_page.html:63]: We don't know the final url of the graph QL url as yet. Think it will open in the same window.

**Review by TNA-Allan — APPROVED** [review]
Thanks @JohnHeeryTNA

**Pull request merged**